### DISCUSSION TITLE
intro

### DISCUSSION THREAD

**Discussion by ghous-mohammad** [community/community#62073] [GitHub Education]
### Hi everyone,

hi everyone

**Comment by drdixit** [comment]
👋Hello @ghous-mohammad

**Reply by ghous-mohammad** [reply]
Hi, how are you?

**Reply by drdixit** [reply]
Doing well, thanks for asking! @ghous-mohammad

**Comment by Siddharthh39** [comment]
Hello Everyone, Hope you all having a nice day.

**Comment by n4ryn** [comment]
Hello @ghous-mohammad
